src/app/dataset/workflow/workflow-header/workflow-header.component.ts

```diff
@@ -157,20 +158,25 @@ export class WorkflowHeaderComponent implements AfterViewInit {
       const n = this.ghost.nativeElement.cloneNode();
       n.style.border = '3px solid #71c07b';
       n.style.boxSizing = 'border-box';
-      n.style.top = '-1000px';
+      n.style.left = '-10000px';
       n.style.transform = 'scale(1)';
       n.style.backgroundSize = 'contain';
       n.style.height = '40px';
+      n.style.position = 'relative';
       n.style.width = '40px';
       document.body.appendChild(n);
       e.dataTransfer.setDragImage(n, 20, 20);
+      this.ghostClone = n;
```

Comment:
Instead of using temporary variable n, why not use this.ghostClone directly?

Reply:
Short answer: we'd need to upgrade typescript!